docs/source/module_configuration/staticsite.rst

```diff
@@ -111,3 +111,149 @@ Most of these options are not required, but are listed here for reference::
               build_output: dist  # overrides default directory of top-level path setting
         regions:
           - us-west-2
+
+`Authorization@Edge`
```

Comment:
should this be changed to `Auth@Edge` for consistency?

Reply:
Corrected